src/wp-includes/rest-api/endpoints/class-wp-rest-pattern-directory-controller.php

```diff
@@ -337,6 +328,11 @@ public function get_collection_params() {
 			'minimum'     => 1,
 		);
 
+		$query_params['slug'] = array(
+			'description' => __( 'Limit results to those matching a pattern (slug).' ),
+			'type'        => array( 'string', 'array' ),
```

Comment:
We should make this just an `array`. The REST API will accept a string value and upgrade it to an array. That way the controller can always treat the `slug` parameter as an `array`.

Reply:
Good to know. Thank you, @TimothyBJacobs.
Fixed in 5e923bc73da1322a2b64bfaf9fc090abfadfc57b